libs/hdf-converters/src/converters-from-hdf/splunk/reverse-splunk-mapper.ts

```diff
@@ -432,48 +431,64 @@ export class FromHDFToSplunkMapper extends FromAnyBaseConverter {
     }
   }
 
-  async toSplunk(
-    config: SplunkConfig,
-    filename: string,
-    webCompatibility = false
-  ): Promise<string> {
-    let service: splunkjs.Service;
-    if (webCompatibility) {
-      service = new splunkjs.Service(new ProxyHTTP.JQueryHttp(''), config);
-    } else {
-      service = new splunkjs.Service(config);
-    }
+  async toSplunk(config: SplunkConfig, filename: string): Promise<string> {
+    const hostname = config.port
+      ? `${config.scheme}://${config.host}:${config.port}`
+      : `${config.scheme}://${config.host}:8089`;
     logger.info(
-      `Logging into Splunk Service: ${config.host} with user ${config.username}`
+      `Logging into Splunk instance at ${hostname} with user ${config.username}`
     );
-    logger.verbose('Got Execution: ' + filename);
+
+    logger.verbose(`Got execution: ${filename}`);
```

Comment:
Can you update this verbose comment to be something more specific?

Reply:
Would something along the lines of `Found designated file to transfer` be more appropriate?

Reply:
Sure, here's the output for pushing to Splunk (it gives you a log) and pulling from Splunk (it doesn't give a log, but shows the content of the instance instead).

<img width="1769" alt="To Splunk" src="https://github.com/mitre/heimdall2/assets/106095667/a00dab0d-18d4-4404-8770-935efd9537d5">
<img width="1771" alt="From Splunk" src="https://github.com/mitre/heimdall2/assets/106095667/c694da4e-8cb7-46a4-883e-af33f7f8bfcd">